less/components/favorite.less

```diff
@@ -32,7 +32,11 @@
+  }
```

Comment:
I actually externalized these so the browser will load the SVG's directly. In commit d4702d3